app/models/constants/involved_civilian.rb

```diff
@@ -21,7 +21,7 @@ module InvolvedCivilian
       WEAPON_FIREARM, 'Firearm replica', 'Knife, blade, or stabbing instrument', 'Other dangerous weapon'
     ].freeze
 
-    RESISTANCE_TYPES = ['Passive non-compliance', 'Resistance', 'Assaultive', 'Life-threatening'].freeze
+    RESISTANCE_TYPES = ['Passive non-compliance', 'Active Resistance', 'Assaultive', 'Life-threatening'].freeze
```

Comment:
I'd only capitalize the "A" in "Active resistance", for consistency with the other ones.